app-text/html2text/html2text-1.3.2a-r2.ebuild

```diff
@@ -0,0 +1,44 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit toolchain-funcs
+
+DESCRIPTION="HTML to text converter"
+HOMEPAGE="http://www.mbayer.de/html2text/"
+SRC_URI="http://www.mbayer.de/html2text/downloads/${P}.tar.gz
+	http://www.mbayer.de/html2text/downloads/patch-utf8-${P}.diff
+	http://www.mbayer.de/html2text/downloads/patch-amd64-${P}.diff
+"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x86-macos"
+
+PATCHES=(
+	"${FILESDIR}/${P}-compiler.patch"
+	"${FILESDIR}/${P}-urlistream-get.patch"
+	"${DISTDIR}/patch-utf8-${P}.diff"
+	"${DISTDIR}/patch-amd64-${P}.diff"
+)
+
+src_unpack() {
+	unpack ${P}.tar.gz
```

Comment:
Why do you need this?

Reply:
Well it's not strictly needed, but since the package also downloads 2 diff files it would try to unpack those files as well which would result in a warning like: ```unpack patch-utf8-html2text-1.3.2a.diff: file format not recognized. Ignoring.```

Reply:
Your choice. We generally do not consider taht a fatal problerm.

Reply:
Well, in that case, lets remove it :)